helm-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
statefulset would be preferred, to not lose data

Reply:
That is great feedback. Most likely I am closing this PR, and going for https://gimlet.io/docs/onechart-reference, but your comment still is valid.